pkg/route/api/v1/types.go

```diff
@@ -43,6 +43,10 @@ type RouteSpec struct {
 	// be defaulted to Service.
 	To kapi.ObjectReference `json:"to"`
 
+	// AdditionalTos is an extension of the 'to' field. If more than one service needs to be
+	// pointed to, then use this field
+	AdditionalTos []kapi.ObjectReference `json:"additionalTos,omitempty"`
```

Comment:
"What's the point of AdditionalTos without weights? If we want to expose alternates, the only reason to do so is to weight them."

The other use case brought about has been about source ip based switching. You may want a particular set of test-users belonging to an ip subnet to be able to access the alternate service. This is how it looks in haproxy:

```
acl app1_privileged src 192.167.0.0/16
acl app1_generic src 192.168.0.0/16

use backend service1_hosts if app1_privileged 
use backend service2_hosts if app1_generic
```

Same route, multiple services, nothing to do with weights.